### PR TITLE
Completed the project with proper GitHub flow

### PR DESCRIPTION
**Pull request by mohammadumar28**
We (myself and my partner) had already completed the project, but mistakenly we didn't follow the GitHub flow. So this project is only a copy of the original project with proper GitHub flow.

Link to the original project: https://github.com/mohammadumar28/Youtube-Video-Player-Page

Please do ask if you need to know anything.

### CODE REVIEW

**Comment by mohammadumar28** [comment]
* Fixed the video player responsiveness and removed the horizontal scrollbar
* Added description and screenshot in the readme
* Replaced ids with classes

**Comment by elmejdki** [comment]
### Pull Request under review by Zakariae El Mejdki

File: css/style.css [hunk 1]
```diff
@@ -0,0 +1,269 @@
+/* Navbar section */
```
Comment (elmejdki) [inline css/style.css:1]: please remove all the comments for your CSS file, also if there is any in the HTML ones.
image

File: css/style.css [hunk 2]
```diff
@@ -0,0 +1,269 @@
...
+    align-items: center;
+  }
+  
+  .profile-image p {
+    font-family: sans-serif;
+    font-size: .80rem;
+    padding-top: 3px;
+  }
+    
+  /* Thumbails/Recommendations section */
```
Comment (elmejdki) [inline css/style.css:216]: here also you need to remove this comment.

File: index.html [hunk 3]
```diff
@@ -0,0 +1,221 @@
...
+
+<head>
+    <meta charset="utf-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Youtube clone</title>
+    <link rel="stylesheet" href="css/style.css">
+</head>
+
+<body>
+    <!-- Navigation bar -->
```
Comment (elmejdki) [inline index.html:12]: please also remove all comments from this file before requesting another code review.

**Comment by mohammadumar28** [comment]
* Resolved all the errors and warnings in the HTML and CSS
* Removed all inline comments

File: index.html [hunk 4]
```diff
@@ -0,0 +1,221 @@
...
+                        <img src="images/profile.jpg" alt="profile photo" />
+                    </div>
+                </div>
+            </li>
+        </ul>
+    </nav>
+    <!-- Main content -->
+    <main>
+        <div class="video-player">
+          <!-- Video player section -->
```
Comment (HADDADSOHAIB) [inline index.html:65]: Please follow what the previous TSE, delete the inline comment to keep your code clean

File: index.html [hunk 5]
```diff
@@ -0,0 +1,221 @@
...
+                <div class="social-section">
+                    <p class="views">130k views • 3 Jun 2020</p>
+                    <div class="flex">
+                        <div class="flex">
+                            <div class="thumbs flex">
+                                <!-- <svg viewBox="0 0 24 24" preserveAspectRatio="xMidYMid meet" focusable="false" class="grey" style="pointer-events: none; display: block; width: 100%; height: 100%;">
+                                    <g class="style-scope yt-icon">
+                                        <path d="M1 21h4V9H1v12zm22-11c0-1.1-.9-2-2-2h-6.31l.95-4.57.03-.32c0-.41-.17-.79-.44-1.06L14.17 1 7.59 7.59C7.22 7.95 7 8.45 7 9v10c0 1.1.9 2 2 2h9c.83 0 1.54-.5 1.84-1.22l3.02-7.05c.09-.23.14-.47.14-.73v-1.91l-.01-.01L23 10z" class="grey"></path>
+                                    </g>
+                                </svg> -->
```
Comment (HADDADSOHAIB) [inline index.html:83]: This should not be in the code, This is commented code.
- [ ] Comments: It’s better to delete commented code. If you need to review a previous version of your code, use git history ([screenshot](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/general/no_inline_comments.png) from [Pathwright](https://microverse.pathwright.com/library/fast-track-curriculum/69047/path/step/66243643/))

File: index.html [hunk 6]
```diff
@@ -0,0 +1,221 @@
...
+                                <img src="images/like.svg" class="grey" alt="like icon" />
+                                <span class="like-text">27k</span>
+                            </div>
+                            <div class="thumbs flex">
+                                <!-- <svg viewBox="0 0 24 24" preserveAspectRatio="xMidYMid meet" focusable="false" class="grey" style="pointer-events: none; display: block; width: 100%; height: 100%;">
+                                    <g class="style-scope yt-icon">
+                                        <path d="M15 3H6c-.83 0-1.54.5-1.84 1.22l-3.02 7.05c-.09.23-.14.47-.14.73v1.91l.01.01L1 14c0 1.1.9 2 2 2h6.31l-.95 4.57-.03.32c0 .41.17.79.44 1.06L9.83 23l6.59-6.59c.36-.36.58-.86.58-1.41V5c0-1.1-.9-2-2-2zm4 0v12h4V3h-4z" class="grey"></path>
+                                    </g>
```
Comment (HADDADSOHAIB) [inline index.html:91]: Same here

**Comment by mohammadumar28** [comment]
* Removed all the comments in the HTML and CSS.

**Pull request merged**